### PR TITLE
Consolidates the notifications panes into a single box

### PR DESCRIPTION
**Pull request by saurav-c**
## Describe your changes and why you are making these changes
This PR consolidates the notification panes into a single box since we no longer have a distinction between org notifications and user notifications.

## Related issue number (if any)
ENG 1610

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

Demo: https://www.loom.com/share/849d3456fce0434d813dcda9b716c757

### CODE REVIEW

File: src/ui/common/src/components/notifications/NotificationsPopover.tsx [hunk 1]
```diff
@@ -40,35 +31,16 @@ export const NotificationsPopover: React.FC<NotificationsPopoverProps> = ({
...
-        return true;
-      } else if (activeTab === NotificationsTabs.Workflow) {
-        return (
-          association === NotificationAssociation.Workflow ||
-          association === NotificationAssociation.WorkflowDagResult
-        );
-      }
-
-      return true;
+      return notification.level !== NotificationLogLevel.Success;
```
Comment (saurav-c) [inline src/ui/common/src/components/notifications/NotificationsPopover.tsx:43]: @vsreekanti This logic was there from before where we only show non-success notifications. Wanted to confirm that this what we want?

**Comment by agiron123** [comment]
So this works for the case of our OSS edition. How do we plan on having separate code for the Enterprise edition? Or is that not a priority for now? 
cc @vsreekanti

File: src/ui/common/src/components/notifications/NotificationsPopover.tsx (hunk 1 above)
Comment (agiron123) [reply src/ui/common/src/components/notifications/NotificationsPopover.tsx:43]: Showing success notifications turned out to be a little too noisy, particularly when users had scheduled workflows, so I believe this is what we want.

**Review by agiron123 — APPROVED** [review]

File: src/ui/common/src/components/notifications/NotificationsPopover.tsx (hunk 1 above)
Comment (vsreekanti) [reply src/ui/common/src/components/notifications/NotificationsPopover.tsx:43]: Yep, I think that's good for now. We can maybe make this an app setting in the future.

**Comment by vsreekanti** [comment]
Good question @agiron123. I'm not super worried about it for the time being. I think in the future, we may want to separate out notifications and events in some form, as we've discussed in the past, so I think this is something that we'll revisit sooner rather than later. For the time being, I think simplifying this view is a win though.

**Comment by agiron123** [comment]
@vsreekanti Sounds good, let's merge as is then. :shipit:

**Pull request merged**